lib/web_ui/lib/initialization.dart

```diff
@@ -82,7 +82,7 @@ Future<void> webOnlyWarmupEngine({
   // Should the app "autoStart"?
   bool autoStart = true;
   if (engine.flutter != null && engine.loader != null) {
-    autoStart = engine.didCreateEngineInitializer == null;
+    autoStart = engine.loader!.isAutostart;
```

Comment:
```suggestion
    autoStart = engine.loader?.isAutostart ?? false;
```

Since anything in javascript can be null at any time, the above is a little bit safer, rather than using `!`

Reply:
But this isn't a JavaScript value. This is the result of `hasProperty` which is explicitly a dart boolean.

Reply:
Ah, gotcha, I totally missed the null check literally above this line (x_X)